client/lib/warn/index.js

```diff
@@ -1,14 +1,13 @@
 /**
  * Internal Dependencies
  */
-var config = require( 'config' );
+import config from 'config';
 
-function warn() {
-	if ( config( 'env' ) !== 'production' ) {
```

Comment:
`NODE_ENV` is set in during the build process and will replaced by `'production' !== 'production'`, which will be removed by uglifier.